riptide-stream/src/main/java/org/zalando/riptide/stream/Streams.java

```diff
@@ -111,11 +110,9 @@
     /**
      * Create default stream converter.
      *
-     * @deprecated use {@link #streamConverter(ObjectMapper)} or {@link #streamConverter(ObjectMapper, List)}
      * @return default stream converter.
      */
-    @Deprecated
-    public static HttpMessageConverter<?> streamConverter() {
+    public static <T> StreamConverter<T> streamConverter() {
```

Comment:
Yes, I think offering this could be useful. We can always deprecate it again, if we decide to do so.